.github/workflows/test.yml

```diff
@@ -54,25 +54,19 @@ jobs:
       fail-fast: false
       matrix:
         name: [
-          "windows-py38",
-          "windows-py38-pluggy",
           "windows-py39",
           "windows-py310",
           "windows-py311",
           "windows-py312",
           "windows-py313",
 
-          "ubuntu-py38",
-          "ubuntu-py38-pluggy",
```

Comment:
I believe the lowest python version marked ones need to move to the next lowest python version

Reply:
So "ubuntu-py39-pluggy", etc. ?

Reply:
Done in [95daa7c](https://github.com/pytest-dev/pytest/pull/12875/commits/95daa7cb47b370d3fa610acb33ac88ff56b79a43)